includes/widgets/widget-instagram.php

```diff
@@ -76,192 +27,36 @@ public function __construct() {
 		 * @param array $instance Saved values from database.
 		 */
 		public function widget( $args, $instance ) {
-			if ( $this->get_cached_widget( $args ) ) {
+			if ( ! current_user_can( 'administrator' ) ) {
```

Comment:
```suggestion
			if ( ! current_user_can( 'manage_options' ) ) {
```

Lets use `manage_options` here.

Reply:
Done